vnext/Microsoft.ReactNative/JsiApi.h

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "JsiError.g.h"
+#include "JsiPreparedJavaScript.g.h"
+#include "JsiRuntime.g.h"
+#include <unordered_map>
+#include "winrt/Microsoft.ReactNative.h"
+
+// facebook::jsi::Runtime hides all methods that we need to call.
+// We "open" them up here by redeclaring the private and protected keywords.
```

Comment:
>redeclaring [](start = 29, length = 11)

wow, that is super hacky... Can't we do friend, inheritance, or some other way?

Reply:
It is not our code and we cannot change it. We can probably use some alternative approach, but there is too much overhead. I do not think that it is worth it. I will consider using a derived type though...

---
In reply to: [460196409](https://github.com/microsoft/react-native-windows/pull/5461#discussion_r460196409) [](ancestors = 460196409)